frontend/packages/process-editor/src/components/ConfigPanel/EditDataType/EditDataType.tsx

```diff
@@ -1,28 +1,32 @@
 import React, { useEffect, useState } from 'react';
-import { useBpmnContext } from '../../../../contexts/BpmnContext';
-import { useBpmnApiContext } from '../../../../contexts/BpmnApiContext';
 import { StudioProperty } from '@studio/components';
 import { useTranslation } from 'react-i18next';
 import { LinkIcon } from '@studio/icons';
 import { SelectDataType } from './SelectDataType';
 import classes from './EditDataType.module.css';
+import { useBpmnContext } from '../../../contexts/BpmnContext';
 
-export const EditDataType = () => {
+export type EditDataTypeProps = {
+  datamodelIds: string[];
+  connectedTaskId: string;
+  existingDataTypeForTask: string | undefined;
+  hideSelectDeleteButton?: boolean;
```

Comment:
Why is this prop include `...Select...` in the name? 🫣

Reply:
Haha, good question. Might have been written on a late evening 😅

Reply:
`hideDeleteButton` is nice 😎